src/dapp/libexec/dapp/dapp-create

```diff
@@ -55,9 +55,8 @@ address=$(set -x; seth send --create "$bin" "${type/constructor/${1#*:}}" "${@:2
 
 [[ $DAPP_VERIFY_CONTRACT ]] && {
   echo >&2 "Verifying contract at $address"
-  path=$(sed 's|\/|/|g' "$DAPP_JSON" | sed "s/.*\"\(.*:${1}\)\".*/\1/g")
   sleep 5
```

Comment:
I wonder why one would sleep here

Reply:
Because it takes time to propagate the block on etherscan. The better way would be to poll etherscan for contract existence (if they have such an API) and then submit for verification.

Reply:
ok!